lib/modules/manager/metadata.spec.ts

```diff
@@ -1,40 +1,51 @@
 import fs from 'fs-extra';
+import { join } from 'upath';
+import { getCustomManagerList } from './custom';
 
 describe('modules/manager/metadata', () => {
   const managerList: string[] = fs
     .readdirSync(__dirname, { withFileTypes: true })
     .filter((dirent) => dirent.isDirectory())
     .map((dirent) => dirent.name)
-    .filter((name) => !name.startsWith('__') || name === 'custom')
+    .filter((name) => !name.startsWith('__') && name !== 'custom')
```

Comment:
Are you sure that you want to flip this?

Reply:
Yes, I have planned to add readme for custom manager in a separate PR where we move from `regexManagers -> customManagers`, so this test should not try to validate the custom manager's readme file as it will be undefined.